Headers/wayland/WaylandServer.h

```diff
@@ -5,7 +5,8 @@
    Copyright (C) 2020 Free Software Foundation, Inc.
 
    Author: Sergio L. Pascual <slp@sinrega.org>
-   Date: February 2016
```

Comment:
You should leave the old dates in, to show when the contribution had happened.

Reply:
👍